core/src/no/group15/playmagic/ecs/systems/MovementSystem.kt

```diff
@@ -116,4 +120,14 @@ class MovementSystem(
 		}
 	}
 
+
+	override fun receive(signal: Signal<CollisionEvent>, event: CollisionEvent) {
```

Comment:
Is this related to movement? maybe bomb system or entity management

Reply:
I put this code here because I need to access the gameMap :grin:

Reply:
It is also in EntityManagementSystem, at least in master

Reply:
Moved to EntityManagementSystem.